Library/GeneralImage.cpp

```diff
@@ -296,6 +297,64 @@ bool GeneralImage::LoadImage(const std::wstring& imageName)
 	return false;
 }
 
+bool GeneralImage::LoadImageFromPluginMeasure(MeasurePlugin* mPlugin)
+{
+	if (!m_Skin || mPlugin == nullptr)
+	{
+		DisposeImage();
+		return false;
+	}
+
+	ImageOptions info;
+	info.m_Path = std::wstring{ L"measure://" } + mPlugin->GetSkin()->GetFilePath() + L"/" + mPlugin->GetName();
+	info.m_FileSize = 0;
+
+	PluginImageData imageData {0};
+	bool success = mPlugin->GetImageData(imageData);
+	if (!success)
+	{
+		DisposeImage();
+		return false;
+	}
+
+	info.m_FileTime = imageData.timestamp;

```

Comment:
If `timestamp` was removed, can the system clock be used for this?

Reply:
It depends on how it is used.
I didn't dig into the image cache works. If the timestamp is just for not reading the drive when file is not updated, then it will be fine to use just `clock.now()` or something.
If the timestamp is also used for not redrawing the same image, or not remaking some transformations with the same image, etc., then removing it will cause higher CPU/GPU load.

Reply:
Oh, judging by line 351 below, the cache is indeed used to optimize transformations on image.